packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -1081,9 +1081,12 @@ class HeapTreeViewState extends State<HeapTree>
 
   void _filter() {
     MemoryScreen.gaAction(name: 'SnapshotFilterDialog');
+    // Dialog isn't dismissed by clicking outside the dialog (modal).
+    // Pressing either the Apply or Cancel button will dismiss.
```

Comment:
is this what we want? I would think clicking outside the dialog should be treated the same as clicking Cancel

Reply:
I think it should be cancel too, but this seemed the easiest for now.  I'll add a todo to dismiss by clicking outside dialog.